frontend/packages/topology/src/components/dropdowns/ApplicationSelector.tsx

```diff
@@ -30,6 +31,7 @@ const ApplicationSelector: React.FC<ApplicationSelectorProps> = ({
   const [nameField] = useField(subPath ? `${subPath}.application.name` : 'application.name');
   const { setFieldValue, setFieldTouched } = useFormikContext<FormikValues>();
   const [applicationExists, setApplicationExists] = React.useState<boolean>(false);
+  const applicationNameInputRef = useRef(null);
```

Comment:
We usually do it like this because `* as React` is already imported in the scope.
```suggestion
  const applicationNameInputRef = React.useRef(null);
```

Reply:
And you can also add a type here so that the call `current?.focus()` is typesafe. An initial value is optional btw.

```suggestion
  const applicationNameInputRef = React.useRef<HTMLInputElement>();
```